omniscidb/QueryEngine/RelAlgExecutionUnit.cpp

```diff
@@ -0,0 +1,68 @@
+#include "RelAlgExecutionUnit.h"
```

Comment:
The license header is missing.

Reply:
Added, check please, I am not sure that I am using correct one. We need to align it along all project.